examples/quijote_inference.py

```diff
@@ -1,13 +1,25 @@
-import os
-import numpy as np
+import argparse
+
 from ili.inference.runner_sbi import SBIRunner
 from ili.validation.runner import ValidationRunner
 
 
-# train a model to infer x -> theta. save it as toy/posterior.pkl
-runner = SBIRunner.from_config("configs/sample_quijoteTPCF_config.yaml")
-runner()
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(
```

Comment:
Instead of having ```use_mdn''' as a parsed argument, I would have the path to the config file as argument, with configs/sample_quijoteTPCF_config.yaml as default. Then the user can use any config file and vary anything they want

Reply:
Added this functionality! Now with quijote_inference.py, you can specify the configuration paths with --cfgtrain and --cfgval .

Reply:
Yeah good idea! I think we just have to keep in mind not to make the example scripts too complex over time since they should be light-weight and easy to run.